README.md

```diff
@@ -228,7 +228,27 @@ export DAEMON_RESTART_AFTER_UPGRADE=true
 cosmovisor start --x-crisis-skip-assert-invariants --home $VAL_2_CHAIN_DIR
 ```
 ### Propose upgrade
-The user owns by val2 is a delegator. So user can vote. Since we changed the gov parameters, the delegations this user delegated are far enough for this proposal to pass.
+The account owned by val2 is a delegator that has more than 67% of voting power, so any vote by this account will be enough to make the proposal succeed. The gov parameter for voting time was also reduced to 60 seconds, so after making the proposal be sure to vote quickly. There are two options for the proposal. One is for automatic downloads, which will take the correct binary from the github releases and install it in the correct cosmovisor directory to start running the upgrade at the correct block height. The other method requires you to download the binary manually and put it in the cosmovisor directory. We recommend trying the autodownload but if you want to be sure that the binary downloaded is absolutely correct it is good to do your own checksum validation. Below are instrucitons for both **autodownload** and **manual download**. The auto-download also includes instruciotns in case the download or the checksum fails.
```

Comment:
this delegator does not have more than 67% power, it is when this delegator vote, he can use his delegations (even though those delegation has already delegated to other validators, @dcwalk please confirm this is right, thank you) to make his vote pass, we changed the voting params so that delegator does not need to have a lot staking to pass his vote.